webapp/roller.js

```diff
@@ -474,6 +474,9 @@ function show_cardset(cardset)
 			if (cards[i].id == cardset.obelisk_pile) {
 				$('.xtra',$tmp).append(' <span class="bane_flag"> &mdash; Obelisk Pile</span>');
 			}
+			if (cardset.boon_list && cardset.boon_list.includes(cards[i].id)) {
```

Comment:
.boon_list seems too generic of a name.
Maybe call the property druid_boons instead.